app/models/ability.rb

```diff
@@ -4,27 +4,24 @@ class Ability
   def initialize(user)
     if user
       can :manage, User, id: user.id
-      can :manage, GroupedIssue do |group|
-        user.is_owner_of?(group.website)
-      end
-      can :manage, Issue do |issue|
-        user.is_owner_of?(issue.group.website)
-      end
-      can :manage, Message do |message|
-        user.is_owner_of?(message.issue.group.website)
-      end
+      can :manage, GroupedIssue
+      can :manage, Issue
+      can :manage, Message
       can :manage, Subscriber do |subscriber|
         user.is_owner_of?(subscriber.website)
       end
       can :manage, Website do |website|
         user.is_owner_of?(website)
       end
+      can :read, Website do |website|
+        user.is_user_of?(website)
+      end
       can :create, Website
       can :manage, WebsiteMember do |website_member|
         user.is_owner_of?(website_member.website)
       end
       can :read, WebsiteMember do |website_member|
-        website_member.website.members.include?(user)
+        website_member.website.is_user_of?(user)
```

Comment:
This is wrong. `is_user_of?` is a method that belongs to the user model not the website model.